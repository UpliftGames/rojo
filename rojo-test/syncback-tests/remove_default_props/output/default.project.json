{
  "name": "remove_default_props",
  "tree": {
    "$className": "DataModel",
    "ReplicatedStorage": {
      "AdjacentMetadataScript": {
        "$path": "AdjacentMetadataScript"
      },
      "DirectoryMetadataPart": {
        "$path": "DirectoryMetadataPart"
      },
      "JsonModelPart": {
        "$path": "Part.model.json"
      },
      "ProjectFilePart": {
        "$className": "Part",
        "$properties": {
          "Size": [
            1.0,
            2.0,
            3.0
          ],
          "Transparency": 1.0,
          "CanCollide": false
        }
      }
    }
  }
}
