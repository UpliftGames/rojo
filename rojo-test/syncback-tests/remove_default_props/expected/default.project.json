{
  "name": "remove_default_props",
  "tree": {
    "$className": "DataModel",
    "ReplicatedStorage": {
      "AdjacentMetadataScript": {
        "$path": "AdjacentMetadataScript"
      },
      "DirectoryMetadataPart": {
        "$path": "DirectoryMetadataPart"
      },
      "JsonModelPart": {
        "$path": "Part.model.json"
      },
      "ProjectFilePart": {
        "$className": "Part",
        "$properties": {
          "Transparency": 1.0
        }
      }
    }
  }
}
